trickuri.go

```diff
@@ -43,6 +43,7 @@ var (
 	port      = flag.Int("p", 1270, "port on which to listen")
 	httpsPort = flag.Int("h", 8443, "port on which the HTTPS proxy will listen")
 	directory = flag.String("d", userHomeDir()+"/.config/trickuri", "default directory in which to save certificates")
+	testIndex = flag.String("i", "index.html", "default file location containing test index")
```

Comment:
We should also use the directory as the root for other tests (e.g. the place where the web-feature-tests folder should be), can you make it so we also use this location for that? (And rename it appropriately, maybe as testcaseDirectory).

Reply:
Done